examples/gimbal_device_test/gimbal_device_test.py

```diff
@@ -0,0 +1,167 @@
+"""
+Example dummy gimbal device component, implemented in PyMAVLink.
+At the current time of writing, there are some issues with implementing a dummy
+gimbal device component with the MAVLinkPassthrough module.
+This example uses the libpilot MAVLink component library (https://gitlab.avhs.best/amadoruavs/pilot/pilot).
+
+Originally authored by: Vincent Wang (2020-03-30).
+"""
+import argparse
+import time
+from pilot import Component
+from pilot.util import MavlinkId
+from pymavlink import mavutil
```

Comment:
What? I don't think we want to depend on pymavlink with MAVSDK.

Reply:
@julianoes Definitely not, that was not the point. Sorry! We were simply trying to use pymavlink for the dummy gimbal manager/device as it was easier than MavlinkPassthrough in MAVSDK as well as we encountered some system ID issues. I will remove this before merge, or if you prefer I take it out of tree now, I'm happy to do so!

@bbworld1 might be better to just keep this out of tree

Reply:
I see.